### PR TITLE
misc: add makefile

### PR DESCRIPTION
**Pull request by Farhad-Shabani**
## Description
This streamlines the workflow for developers who wish to examine changes, manage dependencies, and execute multiple commands simultaneously during the development process.

______

### PR author checklist:
- [X] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: .github/workflows/no-std.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
           override: true
       - run: |
           cd ci/no-std-check
-          make check-panic-conflict
+          make check-no-std
```
Comment (rnbguy) [inline .github/workflows/no-std.yaml:42]: was this a typo from before?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.PHONY: help
+
```
Comment (rnbguy) [inline makefile:2]: We should list what tools are required - `cargo hack` and `cargo release`. Maybe we can directly specify in the help messages.

**Review by rnbguy — CHANGES_REQUESTED** [review]
Thanks @Farhad-Shabani

Added two comments.

File: .github/workflows/no-std.yaml (hunk 1 above)
Comment (Farhad-Shabani) [reply .github/workflows/no-std.yaml:42]: Not a typo I'd say. Renamed so for more clarity.

File: makefile (hunk 2 above)
Comment (Farhad-Shabani) [reply makefile:2]: See if [ed05ba6](https://github.com/cosmos/ibc-rs/pull/1071/commits/ed05ba6eeffbac3d70048b87ea39dcfb5bbb843f) works.

Comment (rnbguy) [reply makefile:2]: Wow ! this is better :smile: :+1:

**Review by rnbguy — APPROVED** [review]
Nice ! :sparkles:

**Pull request merged**